### PR TITLE
[WIPTEST] Add Volume Types Collection including related tests

### PR DESCRIPTION
**Pull request by jadhaj**
Purpose or Intent
=================
Add volume 3 tests which uses volume type:

{{ pytest: cfme/tests/openstack/cloud/test_volumes.py::test_create_volume_with_type cfme/tests/openstack/cloud/test_volumes.py::test_edit_volume_with_type cfme/tests/openstack/cloud/test_volumes.py::test_delete_volume_with_type }}

### CODE REVIEW

File: cfme/storage/volume_type.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+import attr
+
+from navmazing import NavigateToSibling, NavigateToAttribute
+from widgetastic_patternfly import BreadCrumb, Button, Dropdown
+
```
Comment (sshveta) [inline cfme/storage/volume_type.py]: Please remove this blank line

File: cfme/storage/volume_type.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+@navigator.register(VolumeType, 'Details')
+class VolumeTypeDetails(CFMENavigateStep):
+    VIEW = VolumeTypeDetailsView
+    prerequisite = NavigateToAttribute('parent', 'All')
+
+    def step(self, *args, **kwargs):
+        try:
+            self.prerequisite_view.entities.get_entity(name=self.obj.name,                                          surf_pages=True).click()
```
Comment (sshveta) [inline cfme/storage/volume_type.py]: Remove blank space please .
    
        def step(self, *args, **kwargs):
        try:
            self.prerequisite_view.entities.get_entity(name=self.obj.name, surf_pages=True).click()
        except ItemNotFound:
            raise VolumeTypeNotFoundError('Volume {} not found'.format(self.obj.name))

Comment (jadhaj) [reply cfme/storage/volume_type.py]: @sshveta Which space ?

Comment (jawatts) [reply cfme/storage/volume_type.py]: @jadhaj on like 111, the formatting is off. https://github.com/ManageIQ/integration_tests/blob/35c0ed921a1c84799adb1c06fbec915771fa0a3c/cfme/storage/volume_type.py#L111

Comment (jadhaj) [reply cfme/storage/volume_type.py]: @sshveta @jawatts Done

File: cfme/storage/volume_type.py [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+
+@navigator.register(VolumeType, 'Details')
+class VolumeTypeDetails(CFMENavigateStep):
+    VIEW = VolumeTypeDetailsView
+    prerequisite = NavigateToAttribute('parent', 'All')
+
+    def step(self, *args, **kwargs):
+        try:
+            self.prerequisite_view.entities.get_entity(name=self.obj.name, surf_pages=True).click()
```
Comment (sshveta) [inline cfme/storage/volume_type.py]: :+1:

**Pull request merged**